File: src/test/java/org/tindalos/principle/infrastructure/service/jdepend/classdependencies/MyJDependRunnerTest.scala
package org.tindalos.principle.infrastructure.service.jdepend.classdependencies

import org.junit.Assert._
import org.junit.Test

class MyJDependRunnerTest {

  @Test
  def className() {
    assertEquals("aaa.MyClass",MyJDependRunner.className("aaa.MyClass"))
    assertEquals("aaa.MyClass",MyJDependRunner.className("aaa.MyClass$InnerClass"))
    assertEquals("aaa.MyClass",MyJDependRunner.className("aaa.MyClass$InnerClass$Again"))
  }

  @Test
  def xxx2() {
    val x = MyJDependRunner.createNodesOfClasses("org.tindalos.principle")
  }

}

File: src/test/java/org/tindalos/guardrails/internal/infrastructure/service/jdepend/classdependencies/MyJDependRunnerTest.java
package org.tindalos.guardrails.internal.infrastructure.service.jdepend.classdependencies;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class MyJDependRunnerTest {

    @Test
    public void className() {
        assertEquals("aaa.MyClass", MyJDependRunner.className("aaa.MyClass"));
        assertEquals("aaa.MyClass", MyJDependRunner.className("aaa.MyClass$InnerClass"));
        assertEquals("aaa.MyClass", MyJDependRunner.className("aaa.MyClass$InnerClass$Again"));
    }

    @Test
    public void xxx2() {
        MyJDependRunner.createNodesOfClasses("org.tindalos.guardrails");
    }
}
